quickstart.adoc

```diff
@@ -17,15 +17,9 @@ This document describes getting a server and clients up and running for the firs
 Download the installation script e.g. using wget:
 
 ---------------------------------------------------------------------------------------
-wget https://googledrive.com/host/0B1wsLqFoT7i2c3F0ZmI1RDJlUEU/install_script_ubuntu.sh
+wget https://github.com/google/grr/blob/master/install_script_ubuntu.sh
```

Comment:
We already have that script in our repository, I suggest we use it directly instead of adding it a second time. Also, you probably want to use raw for downloading it. This link should work out of the box and make 
https://github.com/google/grr/pull/101 unnecessary:

https://raw.githubusercontent.com/google/grr/master/scripts/install_script_ubuntu.sh